### DISCUSSION TITLE
Popular zigbee wall modules unsupported?

### DISCUSSION THREAD

**Discussion by MrGibbage** [Koenkk/zigbee2mqtt#15647] [Devices]
Some of these are unsupported??
https://www.amazon.com/gp/product/B08K45W4LR

I’ve bought these five times and have not had any issues using them with HA (zha). This weekend I switched to Z2M along with a new sonoff, and now I have four of these that Z2M says are “unsupported”. Seems really strange. I updated the sonoff firmware just this weekend too. Six other similar devices added just fine. It’s just that I have four of them that Z2M doesn’t want anything to do with. One of these newly failed devices was installed when I was using zha and it was working fine. The other three were still in the box, and I just tried installing them this weekend. They were all ordered in the last year.

Z2M identifies these four failed devices as “SWITCH-ZR03-1”, where the ones that were recognized were identified as “SA-03-zigbee”.

Here’s the log output:

Warning 2022-12-18 17:27:35Device ‘0x7cb94c63237a0000’ with Zigbee model ‘SWITCH-ZR03-1’ and manufacturer name ‘eWeLink’ is NOT supported, please follow [Support new devices | Zigbee2MQTT](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html)

Warning 2022-12-18 17:27:39Received message from unsupported device with Zigbee model ‘SWITCH-ZR03-1’ and manufacturer name ‘eWeLink’

Warning 2022-12-18 17:27:39Please see: [Support new devices | Zigbee2MQTT](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html)

I am hesitant to go through the steps for the new device support since these really are supported when they are properly identified. Maybe some of these have different firmware and we are just seeing them? Certainly these arean't all just malfunctioning, but that is a possibility, however remote.

Any ideas?
![51Pim-MsA-L _SL1500_](https://user-images.githubusercontent.com/1330102/208430854-b1a98e7f-4302-4447-8680-89f560fa47bf.jpg)

**Comment by Koenkk** [comment]
I cannot find a supported device with model `SA-03-zigbee`, can you provide the exact model number as detected by z2m?

**Comment by MrGibbage** [comment]
https://www.zigbee2mqtt.io/devices/SA-003-Zigbee.html#ewelink-sa-003-zigbee

I have six of these working in my house right now.

Friendly name/Family Room Reading LampDescription

Last seen/1 minute ago
Availability/Online
Device type/Router
Zigbee Model/SA-003-ZigbeeZigbee
Manufacturer/eWeLink
Description/Zigbee smart plug
Support status/Supported
IEEE Address/0x00124b0022d5fa55
Network address/0x5E63
Firmware build date/20201207
Firmware version/1.0.5
Manufacturer/eWeLink Power
<https://www.zigbee2mqtt.io/supported-devices/#v=eWeLink>ModelSA-003-Zigbee
<https://www.zigbee2mqtt.io/devices/SA-003-Zigbee.html#ewelink-sa-003-zigbee>
Interview completed/True

**Reply by Koenkk** [reply]
Added the `SWITCH-ZR03-1`, will be detected as `SA-003-Zigbee`

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by MrGibbage** [comment]
I am using the regular Z2M right now. If I make a backup, will I be able to delete that addon and then add the Z2M Edge, and then restore the backup? I am assuming the backups are interchangeable between the two versions?
